client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -89,6 +105,12 @@ class TwoFactorActions extends Component {
 							{ translate( 'Continue with your authenticator\u00A0app' ) }
 						</Button>
 					) }
+
+					{ isBackupCodeAvailable && (
+						<Button data-e2e-link="2fa-otp-link" onClick={ this.recordBackupLinkClick }>
```

Comment:
Are we going to specifically need the `data-e2e-link` data attribute? If that's not certain, let's omit it for now.

Reply:
Not really, removed it in 17fd7c3c25